File: week6/sentimental-mario-less/mario.py
import cs50

while True:
    n = cs50.get_int("Height: ")
    if (n < 1 or n > 8):
        continue
    else:
        break


for i in range(1, n+1):
    for j in range(1, n+1-i):
        print(" ", end="")

    for k in range(i):
        print("#", end="")

    print()

File: week1/problem_set/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Ask height within range : 1 - 8 (do while)
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    //Do a left aligned pyramid
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n - i; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= (n - (n - i)); k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
